pkgs/top-level/python-packages.nix

```diff
@@ -2087,6 +2093,8 @@ in {
 
   boltztrap2 = callPackage ../development/python-modules/boltztrap2 { };
 
+  booleanOperations = callPackage ../development/python-modules/booleanOperations { };
```

Comment:
Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)